_install_guide/auth.md

```diff
@@ -208,7 +208,7 @@ See [Spinnaker's docs](https://www.spinnaker.io/setup/security/authorization/sam
 
 
 
-## X509
+## X.509
```

Comment:
I'd prob not change this, there's too many references to `x509` and even configs use `x509`

Reply:
Sounds good, let me revert that